ppr-ui/src/components/mhrRegistration/YourHome/OtherInformation.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <v-card flat class="white pb-6 pt-6 pr-10 pl-8 rounded">
+      <v-row>
+        <v-col cols="2">
+          <label class="generic-label ml" for="">Other</label>
```

Comment:
What is `ml` short for. Is that a class defined locally or a helper?

Can probably remove `for=""` if its not required

Reply:
Sorry i don't follow. Is the `for` prop being used?

Reply:
It is normally used for accessibility and better user experience, when clicking on the label cursor jumps to the respective input field.

Reply:
Oh interesting, i did not know that!
Even without a value provided to the prop? (ie an id)
`for=""` alone does this? How does it know what input to jump too 🤔

Reply:
`for` attr should have/point to an id of the input field. I've added them for each field.